linz-data-importer/linz_data_importer.py

```diff
@@ -428,7 +428,7 @@ def run(self):
 
         if not self.services_loaded:
             if not self.api_key_instance.getApiKeys():
-                self.dlg.uLabelWarning.setText('ERROR: No API Details Provided - see settings')
+                self.dlg.uLabelWarning.setText('To access data add your API details in "Settings". See "Help" for more information.')
```

Comment:
Should "API details" maybe be "API key"?

Reply:
Changed to "API key". Thanks.